indico/modules/events/registration/templates/management/regform_display.html

```diff
@@ -0,0 +1,173 @@
+                {{form.title}}
+            </li>
+            {% endfor %}
+        </ul>
+        <p>
+            {% trans -%}
+                By dragging the column titles, you can choose which fields to
+                display and reorder them.
+            {%- endtrans %}
+            {% trans -%}
+                Custom fields added to registration forms can only be shown if the
+                registration forms are not merged.
+            {%- endtrans %}
+        </p>
+        <div class="i-box titled sortable-column-preview">
+            {{ sortable_lists(_("Shown columns"), enabled_columns,
+                              _("Hidden columns"), disabled_columns) }}
+        </div>
+    </div>
+
+    <div id="registration-forms">
+        <p>
+            {% trans -%}
+                By dragging the form titles, you can choose which to display and reorder them.
+            {%- endtrans %}
+        </p>
+        <div class="i-box titled sortable-form-preview">
+            {{ sortable_lists(_("Shown registration forms"), enabled_forms,
+                              _("Hidden registration forms"), disabled_forms,
+                              form_actions_macro) }}
+        </div>
+    </div>
+
+    <button class="i-button big" type="submit">{% trans %}Save{% endtrans %}</button>
+    <a class="i-button big" href="{{ url_for('.manage_regform_list', event) }}">
+        {% trans %}Back{% endtrans %}
+    </a>
+
+    {{ form_footer(form) }}
+
+    <script>
+        (function () {
+            'use strict';
+            function update() {
+                var mergeForms = $('#merge-forms').prop('checked');
+                $('#participant-list-columns').toggle(mergeForms);
+                $('#registration-forms').toggle(!mergeForms);
+            }
+
+            update();
+            $('#merge-forms').on('change', update);
+
+            $('.sortable-column-preview ul').sortable({
+                connectWith: '.sortable-column-preview ul',
+                placeholder: 'i-label sortable-item placeholder',
+                forcePlaceholderSize: true
+            });
+
+            $('.sortable-form-preview ul').sortable({
+                connectWith: '.sortable-form-preview ul',
+                placeholder: 'i-label sortable-item placeholder',
+                forcePlaceholderSize: true
+            });
+
+            var $form = $('#js-participant-display-form');
+            $form.on('submit', function() {
+                var mergeForms = $('#merge-forms').prop('checked');
+                var data = {
+                    'participant_list_columns': [],
+                    'participant_list_forms': [],
+                    'merge_forms': mergeForms
+                };
+                if (mergeForms) {
+                    $form.find('#publish-registrations input:checked').each(function() {
+                        window.console.log($(this).attr('id'));
```

Comment:
it's not needed. `window` is the global object so any attribute of `window` is also available as a global variable (and vice versa)